VoteStakeRegistry/components/Account/LockTokensAccount.tsx

```diff
@@ -44,7 +44,10 @@ interface DepositBox {
 }
 const unlockedTypes = ['none']
 
-const LockTokensAccount = ({ tokenOwnerRecordPk }) => {
+const LockTokensAccount: React.FC<{
+  tokenOwnerRecordPk: string | string[] | undefined
```

Comment:
(not your fault, dont bother fixing) i appreciate that `tokenOwnerRecordPk` has many possible types, none of which are a PublicKey.